src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/AnalyzerAssemblyService.cs

```diff
@@ -0,0 +1,204 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Reflection;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies
+{
+    [Export(typeof(IAnalyzerAssemblyService))]
+    [AppliesTo(ProjectCapability.DependenciesTree)]
```

Comment:
AppliesTo doesn't apply to single services - only to many services.

Reply:
This also gives the impression that this service lives in a project-system scope, it doesn't; it lives in the VS global scope.

Reply:
@davkean So basically I should just drop the `AppliesTo` attribute?

Reply:
Yes, it can be removed.